src/main/java/organice/logic/commands/MatchCommand.java

```diff
@@ -24,6 +24,10 @@
             + "Parameters: ic/(NRIC) (to match a patient of specified NRIC)";
 
     public static final String MESSAGE_SUCCESS = "Matched all patients and donors";
+    public static final String MESSAGE_SUCCESS_MATCH_PATIENT = "Matched all donors with the patient with Nric:"
```

Comment:
Suggestion: Change to `"Found x matching donors for patient: NAME (NRIC)"`

Reply:
Amended